### PR TITLE
docker-base:light

### PR DESCRIPTION
**Pull request by MBeijer**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -50,6 +50,10 @@ RUN dpkg --add-architecture i386 && \
         libstdc++6:i386 \
         libsdl1.2-dev
 
+RUN rm -rf /var/lib/apt/lists/*
+
+RUN apt clean
```
Comment (apiraino) [inline Dockerfile:55]: According to the documentation, official Debian/Ubuntu images are already this run
https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#apt-get

so I think this is not needed

File: Dockerfile [hunk 2]
```diff
@@ -50,6 +50,10 @@ RUN dpkg --add-architecture i386 && \
         libstdc++6:i386 \
         libsdl1.2-dev
 
+RUN rm -rf /var/lib/apt/lists/*
```
Comment (apiraino) [inline Dockerfile:53]: :+1:

**Comment by apiraino** [comment]
nice!

File: Dockerfile (hunk 1 above)
Comment (MBeijer) [reply Dockerfile:55]: okay

**Pull request merged**